catroid/src/main/java/org/catrobat/catroid/stage/StageActivity.java

```diff
@@ -118,6 +133,55 @@ public void onCreate(Bundle savedInstanceState) {
 		CameraManager.getInstance().setStageActivity(this);
 	}
 
+	private void setupAskHandler() {
+		messageHandler = new Handler(Looper.getMainLooper()) {
+			@Override
+			public void handleMessage(Message message) {
+				ArrayList<Object> params = (ArrayList<Object>) message.obj;
+
+				if (message.what == ASK_MESSAGE) {
+					showDialog((String) params.get(1), (AskAction) params.get(0));
+				}
+			}
+		};
+	}
+
+	private void showDialog(String question, final AskAction askAction) {
+		pause();
+
+		AlertDialog.Builder alertBuilder = new AlertDialog.Builder(new ContextThemeWrapper(this, android.R.style.Theme_Holo_Dialog));
+		final EditText edittext = new EditText(getContext());
+		alertBuilder.setView(edittext);
+		alertBuilder.setMessage(getContext().getString(R.string.brick_ask_dialog_hint));
+		alertBuilder.setTitle(question + "?");
```

Comment:
IMO the ? should not be concatenated here, but entered by the user.
Reason: Right to left languages, or even e.g. Spanish where a reversed question marks are in the front: "¿ question ?"